config/database.yml

```diff
@@ -1,18 +1,8 @@
-# PostgreSQL. Versions 9.3 and up are supported.
-#
-# Install the pg driver:
-#   gem install pg
-# On macOS with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On macOS with MacPorts:
-#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
-# On Windows:
+# Postgresql
 #   gem install pg
-#       Choose the win32 build.
-#       Install PostgreSQL and put its /bin directory on your path.
 #
-# Configure Using Gemfile
-# gem 'pg'
+#   Ensure the pg gem is defined in your Gemfile
+#   gem 'pg'
 #
 default: &default
   host: localhost
```

Comment:
Funktioniert das? Da steht ja nicht, welche Adapter und DB benutzt wird. Dürfte gar nicht funktionieren. 
Ist aber auch nicht Teil von Feature.

Reply:
die Datei habe ich bearbeitet

den adapter nimmt er bei rails auch, wenn er unter dem host definiert wird
der zugehörige docker command zum testen, wäre:
```
docker run -it -d -p 5432:5432 -v pgdata:/var/lib/postgres/data -e POSTGRES_PASSWORD="$POSTGRESPROJECT_DATABASE_PASSWORD" --name postgres postgres:13.2-alpine
``` 
(der, den sie uns mal bereit gestellt haben)

Reply:
Das ist soweit klar, aber es fehlt folgende Angabe:

```
default: &default
  adapter: postgresql
  ...
  ```
  
Das funktioniert wahrscheinlich deshalb ohne diese Zeile, weil Rails von Postgres ausgeht oder schlicht mehrere Ports probiert. Da ActiveRecord ja mehrere DBs ansprechen kann. 

Ich würde dennoch diese Zeile hinzufügen, da die Config ohne diese Zeile nicht korrekt ist. Auch, wenn es funktioniert.

Reply:
meinen sie so?
```
default: &default
  host: localhost
  adapter: postgresql
  ...
```
ist es nicht schon so im aktuellen database.yml ?

Reply:
Nein, du kannst doch hier sehen, dass dieser `adapter: postgresql` fehlt. :)

Reply:
ich bemerke es gerade... es wird tatsächlich nicht angezeigt :/ 
(tut mir leid fehler meiner seite, habe bisher nur den aktuellen feature branch angeschaut)

unter der "Files Changed" Übersicht fehlt aber noch um einiges mehr als nur der adapter...
eigentlich sollte database.yml noch "pool", "user" und "password" beinhalten, was aber im feature branch schon vor längerer zeit von mir gepusht wurde und zurzeit auch dort sind... :/ 

nun gut, falls das Problem nach dem mergen immernoch vorhanden ist, wird es direkt gepatched, 

das eigentlich richtige file existiert ja auch noch im main branch wie auch in einigen feature branches und auch im setup branch, kein großes Problem von unserer Seite

Reply:
Doch, das ist ein Problem! Wenn du es jetzt in main-Branch mergen wirst, ist das alles weg, genau wie hier.

Du musst es also vorher, also hier, beheben.